ivy_tests/test_ivy/test_frontends/test_torch/test_tensor.py

```diff
@@ -452,6 +452,52 @@ def test_torch_instance_addbmm_(
     )
 
 
+# addcdiv_
+@handle_frontend_method(
+    class_tree=CLASS_TREE,
+    init_tree="torch.tensor",
+    method_name="addcdiv_",
+    dtype_and_x=helpers.dtype_and_values(
+        available_dtypes=helpers.get_dtypes("numeric"),
+        num_arrays=3,
+        large_abs_safety_factor=2.5,
+        small_abs_safety_factor=2.5,
+        safety_factor_scale="log",
```

Comment:
Safety factor shouldn't be that large, are you noticing inconsistency in computation?

Reply:
I followed the same strategy with the `addcdiv` test, there will be some inconsistencies between backend frameworks on `float` and `complex` tests if I lower the safety factor. You can view a test case below.
```bash
E           AssertionError:  the results from backend tensorflow and ground truth framework torch do not match
E            [-1. nan]!=[-1. -1.] 
E           
E           
E           Falsifying example: test_torch_instance_addcdiv_(
E               dtype_and_x=(['float32', 'float32', 'float32'],
E                [array([-1., -1.], dtype=float32),
E                 array([-1.0000000e+00, -4.2535296e+37], dtype=float32),
E                 array([-1.   , -0.125], dtype=float32)]),
E               value=0.0,
E               init_flags=FrontendMethodTestFlags(
E                   num_positional_args=0,
E                   as_variable=[False],
E                   native_arrays=[False],
E               ),
E               frontend_method_data=FrontendMethodData(ivy_init_module=<module 'ivy.functional.frontends.torch' from '/workspaces/ivy/ivy/functional/frontends/torch/__init__.py'>, framework_init_module=<module 'torch' from '/home/codespace/.local/lib/python3.10/site-packages/torch/__init__.py'>, init_name='tensor', method_name='addcdiv_'),
E               method_flags=FrontendMethodTestFlags(
E                   num_positional_args=0,
E                   as_variable=[False],
E                   native_arrays=[False],
E               ),
E               frontend='torch',
E               on_device='cpu',
E           )
```

Reply:
it seems to be failing because the original framework is returning `nan` but our backend `torch` returning `-1`, we should handle this in the frontend.

Reply:
it seems to be failing because the original framework is returning `Nan` but our backend torch returning -1, we should handle this in the frontend.